service/controller/v2/arm_templates/main.json

```diff
@@ -397,9 +399,11 @@
           "vmssOsImageOffer": {"value": "[parameters('workerNodes')[0].osImage.offer]"},
           "vmssOsImageSKU": {"value": "[parameters('workerNodes')[0].osImage.sku]"},
           "vmssOsImageVersion": {"value": "[parameters('workerNodes')[0].osImage.version]"},
+          "vmssOverprovision": {"value": "true"},
           "vmssVmCustomData": {"value": "[parameters('workerCloudConfigData')]"},
           "vmssLbBackendPools": {"value": []},
-          "vmssVnetSubnetId": {"value": "[reference('virtual_network_setup').outputs.workerSubnetID.value]"}
+          "vmssVnetSubnetId": {"value": "[reference('virtual_network_setup').outputs.workerSubnetID.value]"},
+          "vmssUpgradePolicy": {"value": "Manual"}
```

Comment:
Why this is parametrized?

Reply:
It's just a place holder if we will want to outsource upgrades to Azure. From latest discussions seems that we will do both "upgrade" and "reimage" in operator. So this is not really needed now.